components/dialog/test/dialog-fullscreen.axe.js

```diff
@@ -0,0 +1,44 @@
+import '../dialog-fullscreen.js';
+import '../../button/button.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+describe('d2l-dialog-fullscreen', () => {
+
+	it('closed', async() => {
+		const el = await fixture(html`<d2l-dialog-fullscreen></d2l-dialog-fullscreen>`);
+		await expect(el).to.be.accessible();
+	});
+
+	it('default with title-text', async() => {
+		const el = await fixture(html`<d2l-dialog-fullscreen opened title-text="My Dialog"></d2l-dialog-fullscreen>`);
+		await expect(el).to.be.accessible();
+	});
+
+	it('footer buttons', async() => {
+		const el = await fixture(html`
+			<d2l-dialog-fullscreen title-text="Dialog Title" opened>
+				<div>Some dialog content</div>
+				<d2l-button slot="footer" primary data-dialog-action="done">Done</d2l-button>
+				<d2l-button slot="footer" data-dialog-action>Cancel</d2l-button>
+			</d2l-dialog-fullscreen>
+		`);
+		await expect(el).to.be.accessible();
+	});
+
+	it.skip('no title-text', async() => {
```

Comment:
We currently list `title-text` as optional, but it's an axe violation if not present on this and `d2l-dialog` (`d2l-dialog-confirm` uses `text` as backup). I've asked Jeff what he would recommend here.